modules/nlu/src/backend/intents/intent-service.ts

```diff
@@ -104,26 +104,39 @@ export async function updateIntentsSlotsEntities(
   })
 }
 
-export async function getIntentsTopics(ghost: sdk.ScopedGhostService, intentNames: string[]) {
+/**
+ * This method read every workflow to extract their intent usage, so they can be in sync with their topics.
```

Comment:
What do you mean by  "be in sync with their topic" ? do you mean intent.contexts === allTopicsHavingIntent(intentName) ?

if yes why can we rename this for updateContextsFromTopics ?